core-java-modules/core-java-string-operations-4/src/test/java/com/baeldung/charcomparison/CharComparisonUnitTest.java

```diff
@@ -0,0 +1,65 @@
+package com.baeldung.charcomparison;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Objects;
+
+import org.junit.jupiter.api.Test;
+
+public class CharComparisonUnitTest {
+
+    @Test
+    public void givenTwoPrimitiveChars_whenUsingRelationalOperators_thenCompare() {
+
```

Comment:
The code formatting is OK, however, there´s no need for the first line to be blank. Likewise, the last line of the method should not be a blank one.

Reply:
done